source/configuration_manual/shared_mailboxes/cluster_setup.rst

```diff
@@ -49,7 +49,10 @@ in this mailbox ``someone1`` would access a mailbox with the name
     type = shared
     prefix = shared/%%u/
     list = children
-    location = imapc:
+    # Use with INDEXPVT to enable per user \Seen flags
+    # If running earlier versions than 2.3.15, or if using obox storage
+    # INDEXPVT is not supported
```

Comment:
* Sentences should end with `.`
 * "Use INDEXPVT" without "with"?
 * I wonder if it's per user or per-user. I've usually used per-user, but I do see both with google.